qa/rolling-upgrade/build.gradle

```diff
@@ -9,12 +9,13 @@ apply from : "$rootDir/qa/build.gradle"
 
 String knn_bwc_version = System.getProperty("bwc.version")
 String baseName = "knnBwcCluster-rolling"
+String opensearch_version_upgraded = System.getProperty("rolling.bwctests.opensearch.version", opensearch_version)
```

Comment:
Is it actually possible to specify a different version than current to upgrade to?

Reply:
yes, we need to add it to that matrix